recipes/aioserial/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "aioserial" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "291357a062d537d2122cdc95fe2be2b74eaf310b385b067b70da3cf8b316ae7e"
```

Could you please remove the quotes between the hash?
```suggestion
  sha256: 291357a062d537d2122cdc95fe2be2b74eaf310b385b067b70da3cf8b316ae7e
```